helm/resources/certs/README.md

```diff
@@ -0,0 +1,12 @@
+All certificates required for deployment should be placed here. Required list of certificates with description:
```

Comment:
I don't think we should repeat the same mistake that has happened to the v1 Helm chart, which is requiring to put certificate files into certain folder to be able to deploy it. Instead we should be able to pass these certs as base64 encoded strings.
This ways you can use the Helm chart with the usual commands (`helm repo add`, `helm install`).

Reply:
Well then we may switch to a simple multiline strings and then encode them into base64
I'll make modification so it would work this way

Reply:
Done